packages/manager/apps/pci-block-storage/src/pages/new/New.page.tsx

```diff
@@ -142,13 +153,26 @@ export default function NewPage(): JSX.Element {
             isDisabled: stepper.validation.step.isLocked,
           }}
           subtitle={
-            <OsdsText
-              level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
-              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
-              color={ODS_THEME_COLOR_INTENT.text}
-            >
-              {tAdd('pci_projects_project_storages_blocks_add_type_subtitle')}
-            </OsdsText>
+            <span>
+              <OsdsText
+                level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+                size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+                color={ODS_THEME_COLOR_INTENT.text}
+              >
+                {tAdd('pci_projects_project_storages_blocks_add_type_subtitle')}
```

Comment:
We can take the opportunity to handle a single instance of `t`

Reply:
It would add too much modifications I think

Reply:
All right, but we could consider refactor it some day